### ISSUE TITLE
Switch API key to publish with Jeremy's account

### ISSUE THREAD

**Issue by Aariq** [uace-azmet/azmet-forecast-qa#59]
Followup to #53 
Currently the pipeline ends by updating a pinned dataset published by @Aariq.  Eventually should switch this over to Jeremy?  I think this will require the following:
- [ ] Download the current dataset from ericrscott/fc_daily
- [ ] Have @jeremylweiss publish this dataset to his Posit Connect board
- [ ] Update code to point to the new pin: https://github.com/uace-azmet/azmet-forecast-qa/blob/a9fa0613af50a6ab8b3ba43c13ef275e7ba244e6/R/pin_forecast.R#L11C44-L11C63
- [ ] Switch the GitHub secret for CONNECT_API_KEY to one generated by Jeremy.

**Comment by KristinaRiemer** [comment]
@Aariq I think this already happened?

**Comment by Aariq** [comment]
No, I don't think it did, but I'm not sure Jeremy is really using this forecast-based QA/QC at all.
